src/sage/modular/quasimodform/element.py

```diff
@@ -294,6 +294,45 @@ def __bool__(self):
         """
         return bool(self._polynomial)
 
+    def depth(self):
+        r"""
+        Return the depth of the given quasimodular form.
+
+        Note that the quasimodular form must be homogeneous of weight
+        `k`. Recall that the *depth* is the integer `p` such that
```

Comment:
```suggestion
        Note that the quasimodular form must be homogeneous.
        Recall that the *depth* is the integer `p` such that
```
Then define $k$ somewhere.